### PR TITLE
improve travis

### PR DESCRIPTION
**Pull request by mmckinst**
its still failing on something for mongodb but this is at least better

### CODE REVIEW

File: .puppet-lint.rc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 --fail-on-warnings
---no-80chars-check
+--no-140chars-check
```
Comment (ekohl) [inline .puppet-lint.rc]: This is managed through https://github.com/Katello/foreman-installer-modulesync which most like needs a similar change to https://github.com/theforeman/foreman-installer-modulesync/pull/36

**Comment by ehelms** [comment]
I've opened a PR to update our modulesync [1] and once its merged will fan the change out to our modules.

[1] https://github.com/Katello/foreman-installer-modulesync/pull/8

**Comment by ehelms** [comment]
@mmckinst I think through various PRs we've fixed all the issues here.

**Comment by mmckinst** [comment]
Confirmed. Thanks!

**Pull request closed without merging**